src/olympia/users/models.py

```diff
@@ -637,23 +638,79 @@ def fetch_names():
         return any(n in name for n in blocked_list)
 
 
-@python_2_unicode_compatible
-class UserRestriction(ModelBase):
+class IPAddressUserRestriction(ModelBase):
```

Comment:
We don't need both `IPAddressUserRestriction` and `IPNetworkUserRestriction`, the former is just a special case of the latter. If an IP is entered in the admin we can convert it to a network automatically by just making it a `/32`

Reply:
I kinda thought we wanted to split that up to simplify permission management? We could split it in the admin anyway I think…

Reply:
We do want different permissions for email and ip bans, but there is no need to differentiate between ip and network, or email and domain, respectively.